src/definitions/constants.rs

```diff
@@ -31,4 +31,11 @@ lazy_static! {
     // Value generated from get_selector_from_name('__validate_declare__')
     pub static ref VALIDATE_DECLARE_ENTRY_POINT_NAME: Felt =
         felt_str!("1148189391774113786911959041662034419554430000171893651982484995704491697075");
+    pub static ref TEST_SEQUENCER_ADDRESS: Felt =
+        felt_str!("4096");
+    pub static ref TEST_ERC20_CONTRACT_ADDRESS: Felt =
+        felt_str!("4097");
+    pub static ref TRANSFER_ENTRY_POINT_SELECTOR: Felt =
+        felt_str!("232670485425082704932579856502088130646006032362877466777181098476241604910");
```

Comment:
Maybe add a comment detailing how to generate this number? Check line 31 for an example.